trackpy/framewise_data.py

```diff
@@ -248,13 +248,13 @@ def __init__(self, filename, key='FrameData', mode='a', t_column='frame',
         self._t_column = t_column
         self.store = pd.HDFStore(self.filename, mode, **kwargs)
 
-        with pd.get_store(self.filename) as store:
-            try:
-                store[self.key]
-            except KeyError:
-                pass
-            else:
-                self._validate_node(use_tabular_copy)
+        store = pd.HDFStore(self.filename)
```

Comment:
We're getting a new error in Appveyor (Windows):

```
[00:10:20] ======================================================================
[00:10:20] ERROR: test_storage (trackpy.tests.test_feature_saving.TestPandasHDFStoreSingleNodeCompressed)
[00:10:20] ----------------------------------------------------------------------
[00:10:20] Traceback (most recent call last):
[00:10:20]   File "C:\projects\trackpy\trackpy\tests\test_feature_saving.py", line 87, in test_storage
[00:10:20]     os.remove(STORE_NAME)
[00:10:20] PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'temp_for_testing_5115073464.h5'
[00:10:20] 
[00:10:20] ----------------------------------------------------------------------
```

Windows (or at least the Windows API that Python uses) is much more strict than Unix/Linux regarding simultaneous access, copy-on-write, etc. However it doesn't seem like the changes to `framewise_data.py` would cause this error. Any ideas?

Reply:
The new commit should fix this.